doc/api/assert.markdown

```diff
@@ -409,3 +408,4 @@ assert.throws(
 [`Error`]: errors.html#errors_class_error
 [`RegExp`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
 [`TypeError`]: errors.html#errors_class_typeerror
+[Locked]: documentation.html#documentation_stability_index
```

Comment:
Heh, good catch. Maybe it's better to say not lex but ASCII order? Since usual `str1 < str2` JS check was used.